build/images/Dockerfile.build.windows

```diff
@@ -44,3 +47,10 @@ COPY --from=antrea-build-windows  /antrea/bin/antrea-cni.exe /k/antrea/cni/antre
 
 RUN mkdir C:\k\antrea\utils
 COPY --from=antrea-build-windows /wins/wins.exe /k/antrea/utils/wins.exe
+
+COPY --from=windows-ovs /Windows/System32/vcruntime140.dll /Windows/System32/
+COPY --from=windows-ovs /Windows/System32/libeay32.dll /Windows/System32/
+COPY --from=windows-ovs /Windows/System32/ssleay32.dll /Windows/System32/
+
+RUN mkdir C:\openvswitch\usr\bin
+COPY --from=windows-ovs /openvswitch/usr/bin /openvswitch/usr/bin/
```

Comment:
Can we copy them to `/antrea/bin/` so we can unify `Run-AntreaAgent-Containerd.ps1` to the following?
```
$ErrorActionPreference = "Stop"
$mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
$mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
$env:PATH = $env:PATH + ";$mountPath/k/antrea/bin"
& antrea-agent.exe --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0

```